### PR TITLE
[C-996] Remove NATIVE_MOBILE references

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

- Removes unused NATIVE_MOBILE vars and code branches
- Used `isNativeMobile` for common sagas
- Removed css and other native-specific web components

### CODE REVIEW

File: packages/web/.env/.env.mobile.prod [hunk 1]
```diff
@@ -1,4 +1 @@
-REACT_APP_NATIVE_MOBILE=true
-REACT_APP_NATIVE_NAVIGATION_ENABLED=true
-
-PORT=3012
\ No newline at end of file
+PORT=3012
```
Comment (sliptype) [inline packages/web/.env/.env.mobile.prod:1]: Can probably remove this and the corresponding command entirely

**Review by sliptype — APPROVED** [review]
It just keeps going! Sooo much deleted code, amazing

File: packages/web/.env/.env.mobile.prod (hunk 1 above)
Comment (dylanjeffers) [reply packages/web/.env/.env.mobile.prod:1]: oh yeah totes!!!

Comment (raymondjacobson) [inline packages/web/.env/.env.mobile.prod]: we can actually delete this file entirely

File: packages/web/src/common/store/oauth/sagas.ts [hunk 2]
```diff
@@ -296,8 +293,7 @@ function* watchInstagramAuth() {
 }
 
 const sagas = () => {
-  const sagas = [watchTwitterAuth, watchInstagramAuth]
-  return NATIVE_MOBILE ? sagas.concat(mobileSagas()) : sagas
+  return [watchTwitterAuth, watchInstagramAuth]
```
Comment (raymondjacobson) [inline packages/web/src/common/store/oauth/sagas.ts:296]: jeepers jeffers i touched this file too!

**Review by raymondjacobson — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-996-remove-NATIVE_MOBILE

**Review by sliptype — APPROVED** [review]

**Review by nicoback2 — APPROVED** [review]
its gone!

File: packages/web/src/common/store/oauth/sagas.ts (hunk 2 above)
Comment (dylanjeffers) [reply packages/web/src/common/store/oauth/sagas.ts:296]: word i can help resolve for you. basically i just ignored nativeSagas, expecting that you moved it into mobile package etc

**Pull request merged**